### PR TITLE
Split autoplay GIFs and videos into different settings

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/5771
Type: enhancement

---
Here's what your changelog entry will look like:

## ✨ Features
 * Split autoplay GIFs and videos into different settings ([\#6726](https://github.com/matrix-org/matrix-react-sdk/pull/6726)). Fixes vector-im/element-web#5771 and vector-im/element-web#5771. Contributed by [SimonBrandner](https://github.com/SimonBrandner).

Preview: https://6130f11a4ac20febac0d9a5a--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/settings/handlers/AccountSettingsHandler.ts [hunk 1]
```diff
@@ -110,6 +110,21 @@ export default class AccountSettingsHandler extends MatrixClientBackedSettingsHa
             return content ? content['enabled'] : null;
         }
 
+        // Special case for autoplaying videos and GIFs
+        if (["autoplayGifs", "autoplayVideo"].includes(settingName)) {
+            const settings = this.getSettings() || {};
+            const value = settings[settingName];
+            // Fallback to old combined setting
+            if (value === null || value === undefined) {
+                const oldCombinedValue= settings["autoplayGifsAndVideos"];
```
Comment (Palid) [inline src/settings/handlers/AccountSettingsHandler.ts:119]: Looks like somehow it went through linter on CI.
```suggestion
                const oldCombinedValue = settings["autoplayGifsAndVideos"];
```

**Review by Palid — COMMENTED** [review]
LGTM, but would be nice if we had a proper acceptance from the Product team, @nadonomy what do you think? 😄

**Comment by nadonomy** [comment]
@SimonBrandner thanks! Looks good in Settings. In the Netlify build my test account is showing the toggles as off.

Does this PR impact the default value at all?

**Comment by SimonBrandner** [comment]
> @SimonBrandner thanks! Looks good in Settings. In the Netlify build my test account is showing the toggles as off.
> 
> Does this PR impact the default value at all?

No, it shouldn't, both were off by default

**Review by Palid — APPROVED** [review]
Seems good, merging!

**Pull request merged**